pom.xml

```diff
@@ -310,9 +310,9 @@
     </build>
     <dependencies>
         <dependency>
-            <groupId>javax.ejb</groupId>
-            <artifactId>javax.ejb-api</artifactId>
-            <version>3.2-b01</version>
+            <groupId>jakarta.ejb</groupId>
```

Comment:
Is the larger team ready for this change?  I thought that the EJB and JPA artifacts were prematurely published to maven without having a formal Eclipse review...  Eventually, these type of changes need to be done, so maybe we should just start with these and if something changes, we just do another PR...

Reply:
I agree.  We can start with these and change them if that is needed down the road.

Reply:
+1